### PR TITLE
feat(core): enable tag inputs without dropdowns

### PR DESCRIPTION
**Pull request by TimSielemann**

### CODE REVIEW

File: core/src/components/cat-tag/cat-tag.tsx [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+    if (['Enter', ...this.tagCreationChars].includes(event.key)) {
+      event.preventDefault();
+      if (this.input?.value.trim() && !this.value?.includes(this.input?.value.trim())) {
+        this.value = [...(this.value ?? []), this.input.value.trim()]
+        this.catChange.emit(this.value);
+      }
+      if(this.input) {
+        this.input.value = ''
+      }
+    } else if (['Backspace'].includes(event.key) && this.input?.selectionStart === 0 && (this.value?.length ?? 0) > 0) {
```
Comment (fynnfeldpausch) [inline core/src/components/cat-tag/cat-tag.tsx:145]: isnt this the same:  `(this.value?.length ?? 0) > 0` and `this.value?.length`

File: core/src/components/cat-tag/cat-tag.tsx [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+      this.value = this.value?.slice(0, -1) ?? [];
+      this.catChange.emit(this.value);
+    }
+  }
+
+  @Watch('errors')
+  onErrorsChanged(value?: boolean | string[] | ErrorMap) {
+    console.log('error changed', value, this.errors)
```
Comment (fynnfeldpausch) [inline core/src/components/cat-tag/cat-tag.tsx]: remove

File: core/src/components/cat-tag/cat-tag.tsx [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+  }
+
+  private onInput() {
+    const currentValue =  [...new Set((this.input?.value?.split(this.createSplitRegex(this.tagCreationChars)) ?? []))].filter(value =>
+      !!value &&
+      !this.value?.includes(value)
+    );
+    if (currentValue.length > 1) {
+      this.value = [...(this.value ?? []), ...currentValue]
+      console.log(this.input?.value, this.value);
```
Comment (fynnfeldpausch) [inline core/src/components/cat-tag/cat-tag.tsx:245]: remove

**Review by fynnfeldpausch — CHANGES_REQUESTED** [review]

**Review by fynnfeldpausch — APPROVED** [review]

File: core/src/components/cat-tag/cat-tag.tsx (hunk 1 above)
Comment (TimSielemann) [reply core/src/components/cat-tag/cat-tag.tsx:150]: no, the latter will render a 0 into the dom...

File: core/src/components/cat-tag/cat-tag.tsx (hunk 2 above)
Comment (TimSielemann) [reply core/src/components/cat-tag/cat-tag.tsx]: done

File: core/src/components/cat-tag/cat-tag.tsx (hunk 3 above)
Comment (TimSielemann) [reply core/src/components/cat-tag/cat-tag.tsx]: done

**Pull request merged**